rust/src/tests/chmod/permission.rs

```diff
@@ -79,21 +80,20 @@ fn clear_isgid_bit(ctx: &mut SerializedTestContext) {
     let path = ctx.create(FileType::Regular).unwrap();
     chmod(&path, Mode::from_bits_truncate(0o0755)).unwrap();
 
-    let user = Uid::from_raw(65535);
-    let group = Gid::from_raw(65535);
```

Comment:
Do you know why the original test used 65535 instead of 65534?

Reply:
Not sure, maybe a typo? This UID is present in some other tests and seems to be useful for those, but could be replaced by 65534 for this one.